dev/bin/build.js

```diff
@@ -218,24 +219,42 @@ function ensureFilesExist(directory, files) {
     }
 }
 
-
 /**
  * @param {string[]} argv
  */
 export async function main(argv) {
-    const args = getArgs(argv, new Map(/** @type {[key: string, value: (boolean|null|number|string|string[])][]} */ ([
-        ['all', false],
-        ['default', false],
-        ['manifest', null],
-        ['dry-run', false],
-        ['dry-run-build-zip', false],
-        ['yomitan-version', '0.0.0.0'],
-        [null, []]
-    ])));
-
-    const dryRun = /** @type {boolean} */ (args.get('dry-run'));
-    const dryRunBuildZip = /** @type {boolean} */ (args.get('dry-run-build-zip'));
-    const yomitanVersion = /** @type {string} */ (args.get('yomitan-version'));
+    /** @type {import('util').ParseArgsConfig['options']} */
+    const parseArgsConfigOptions = {
+        'all': {
+            type: 'boolean',
+            default: false
+        },
+        'default': {
+            type: 'boolean',
+            default: false
+        },
+        'manifest': {
+            type: 'string'
+        },
+        'dry-run': {
+            type: 'boolean',
+            default: false
+        },
+        'dry-run-build-zip': {
+            type: 'boolean',
+            default: false
+        },
+        'yomitan-version': {
```

Comment:
`yomitan-version` - can this be made into just `version`? Will probably affect other places in the app/deployment process, but IMO the name of the extension should probably be omitted from a lot of the internal codebase.

Reply:
You can search codebase for regex `dry-run|yomitan-version` to find other files that need adjustment